communications/rfc.rst

```diff
@@ -177,6 +177,15 @@ If an RFC includes API changes, including deprecations, or updates to a data mod
 
 In all of these cases, the RFC is marked as **Flagged** to indicate that it is being considered by the DMCCB.
 
+As a result of the discussion on a **Flagged** RFC, the DMCCB may decide to approve it and move forward with the proposal.
+In this case, the RFC status is set to **Board Recommended**.
+The assignee can then attach the triggering issues and adopt the RFC following the standard process or reopen it changing its status to **Fagged**.
+
+In the case that the DMCCB is not approving the RFC, its status will be set to **Withdrawn**.
+Tn the case that the DMCCB consider that the RFC do not require its approval, the RFC status will be set back to **Proposed**.
```

Comment:
"If the DMCCB declines to consider the RFC, the DMCCB will change the status back to **Proposed**."